src/View/Form/FormContext.php

```diff
@@ -33,17 +35,31 @@ class FormContext implements ContextInterface
      */
     protected $_form;
 
+    /**
+     * Validator name.
+     *
+     * @var string|null
+     */
+    protected $_validator = null;
+
     /**
      * Constructor.
      *
      * @param array $context Context info.
+     *
+     * Keys:
+     *
+     * - `entity` The Form class instance this context is operating on.
+     * - `validator` Optional name of the validation method to call on the Form object.
      */
     public function __construct(array $context)
     {
-        $context += [
-            'entity' => null,
```

Comment:
Is removing this default a BC break or does it already fail further into processing?

Reply:
It's not a BC break, even currently you would have got an error trying to use any of its methods if entity wasn't set. I added an explicit check and exception now.